src/include/ipc/info.h

```diff
@@ -122,7 +122,7 @@ struct sof_ipc_cc_version {
 
 	char name[16]; /* null terminated compiler name */
 	char optim[4]; /* null terminated compiler -O flag value */
-	char desc[]; /* null terminated compiler description */
+	char desc[24]; /* null terminated compiler description */
```

Comment:
@xiulipan I know that there is no need to make change on kernel side to make it working, but I'd like to have comment in this line, something like:

> /* null terminated compiler description */
> /* flex array from host side */

Because there it will be obvious that there can't be any more field after `desc`

Reply:
still did not get why we will have flex array from host side.
This structure is compiled in ELF file

Reply:
> still did not get why we will have flex array from host side.

Just look at `include/sound/sof/info.h` in kernel repo, there is definition of `sof_ipc_cc_version` from kernel side

Reply:
@xiulipan can you do a PR for kernel too, thanks